atc/db/migration/migrations/1622031307_add_build_comment.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE builds
```

Comment:
Good idea. I've make the `build_id` the primary key of the `build_comments` table, but this might prevent there from being multiple comments per build in the future. Would you like me to change it or is it okay for now?

Reply:
Happy to leave it as is - shouldn't be too hard to migrate if we end up wanting multiple comments per build in the future.